documentation/docs/css/extra.css

```diff
@@ -4,6 +4,16 @@
     top: 0.6rem;
     left: 0.6rem;
   }
+  /* Modify rendering of numbered lists in PDF */
```

Comment:
It's a nice fix. However, there are two unsolved issues in it: 
1/ the last curly brace, at line 35, is breaking this stylesheet
2/ all of the styles in this file were meant to go inside of the `@media print` statement.